_data/projects/VisitMeet.yml

```diff
@@ -0,0 +1,11 @@
+name: VisitMeet
+desc: VisitMeet is a cheapeast way to travel and develop 3rd world countries at the same time.
+site: https://github.com/VisitMeet/visitmeet
+tags:
+- oss
+- web
+- ruby on rails
+- javascript
+upforgrabs:
+  name: VisitMeet
```

Comment:
This value should be the name of the label you're using to tag issues which are available to new contributors.